README.rst

```diff
@@ -44,7 +45,7 @@ Here's a brief example of how the package works:
 
     >>> from pprint import pprint
     >>> from ansys.grantami.bomanalytics import Connection, queries
-    >>> cxn = Connection(servicelayer_url='http://localhost/mi_servicelayer').with_autologon().connect()
```

Comment:
I think this should, if possible, match the documentation. There we leave the keyword but add a line break before `.with_autologon()`. Either it should be that, or we should change the documentation to match.

Reply:
My motivation in doing it this way was to show the simplest possible code.

Having the line break syntax makes it look more complicated than it actually is to create a connection if someone's running in Windows and wants to authenticate with Kerberos, which is going to be the case for probably 80%+ of the people using this package.

It's a bit inconsistent, yes, but I think the goals of this code is different. Here we're showing the simplest possible working example to give a _potential_ user of the package a general idea of what the interface looks like. The examples though are to educate an _actual_ user as to how they use the package.

Reply:
OK, I think that makes sense